lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -229,18 +229,18 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of sql queries to default,
         #     # of mongo queries,
         # )
-        ('no_overrides', 1, True, False): (35, 6),
-        ('no_overrides', 2, True, False): (41, 6),
-        ('no_overrides', 3, True, False): (51, 6),
-        ('ccx', 1, True, False): (35, 6),
-        ('ccx', 2, True, False): (41, 6),
-        ('ccx', 3, True, False): (51, 6),
-        ('no_overrides', 1, False, False): (35, 6),
-        ('no_overrides', 2, False, False): (41, 6),
-        ('no_overrides', 3, False, False): (51, 6),
-        ('ccx', 1, False, False): (35, 6),
-        ('ccx', 2, False, False): (41, 6),
-        ('ccx', 3, False, False): (51, 6),
+        ('no_overrides', 1, True, False): (21, 6),
+        ('no_overrides', 2, True, False): (21, 6),
+        ('no_overrides', 3, True, False): (21, 6),
+        ('ccx', 1, True, False): (21, 6),
+        ('ccx', 2, True, False): (21, 6),
+        ('ccx', 3, True, False): (21, 6),
+        ('no_overrides', 1, False, False): (21, 6),
+        ('no_overrides', 2, False, False): (21, 6),
+        ('no_overrides', 3, False, False): (21, 6),
+        ('ccx', 1, False, False): (21, 6),
+        ('ccx', 2, False, False): (21, 6),
+        ('ccx', 3, False, False): (21, 6),
```

Comment:
I second Cliff's comment that we should restructure usages of this data to draw from a common `EXPECTED_QUERY_COUNT` tuple instead of specifying the number on each line.